wasm/jit/jit_amd64.go

```diff
@@ -98,7 +98,9 @@ func (e *engine) compileWasmFunction(f *wasm.FunctionInstance) (*compiledWasmFun
 				return nil, fmt.Errorf("error handling global.set operation: %w", err)
 			}
 		case *wazeroir.OperationLoad:
-			return nil, fmt.Errorf("unsupported operation in JIT compiler: %v", o)
+			if err := builder.handleLoad(o); err != nil {
+				return nil, fmt.Errorf("error handling global.set operation: %w", err)
```

Comment:
ps can we use a string form to generify the error handling across these branches?

```suggestion
				return nil, fmt.Errorf("error handling load operation: %w", err)
```